core/src/main/java/tc/oc/pgm/listeners/ChatDispatcher.java

```diff
@@ -238,6 +238,9 @@ public void sendReply(Match match, Audience audience, MatchPlayer sender, @Text
 
   @EventHandler(ignoreCancelled = true, priority = EventPriority.MONITOR)
   public void onChat(AsyncPlayerChatEvent event) {
+    // Ignore test messages from method "send"
+    if (event.getMessage().contains("8af5859a-1d6e-4fbd-ae5f-4c63ff630fa4")) return;
```

Comment:
This is too hacky, you'll need to develop another way.

Reply:
You could keep a `Cache` of recently sent `AsyncPlayerChatEvent`s.

Reply:
I did a version of that now. Thanks for feedback.